include/mbedtls/aes.h

```diff
@@ -57,68 +60,91 @@ extern "C" {
 #endif
 
 /**
- * \brief          AES context structure
+ * \brief The AES context-type definition.
  *
- * \note           buf is able to hold 32 extra bytes, which can be used:
- *                 - for alignment purposes if VIA padlock is used, and/or
- *                 - to simplify key expansion in the 256-bit case by
- *                 generating an extra round key
+ * The AES context is passed to the APIs called.
+ *
+ * \note           This buffer can hold 32 extra Bytes, which can be used for
+ *                 one of the following purposes:
+ *                 <ul><li>Alignment if VIA padlock is used.</li>
+ *                 <li>Simplifying key expansion in the 256-bit case by
+ *                 generating an extra round key.</li></ul>
  */
 typedef struct
 {
-    int nr;                     /*!<  number of rounds  */
-    uint32_t *rk;               /*!<  AES round keys    */
-    uint32_t buf[68];           /*!<  unaligned data    */
+    int nr;                     /*!< The number of rounds.*/
+    uint32_t *rk;               /*!< AES round keys.*/
+    uint32_t buf[68];           /*!< Unaligned data buffer.*/
 }
 mbedtls_aes_context;
 
 /**
- * \brief          Initialize AES context
+ * \brief          This function initializes the specified AES context.
+ *
+ *                 It  must be the first API called before using
+ *                 the context.
  *
- * \param ctx      AES context to be initialized
+ * \param ctx      The AES context to initialize.
  */
 void mbedtls_aes_init( mbedtls_aes_context *ctx );
 
 /**
- * \brief          Clear AES context
+ * \brief          This function releases and clears the specified AES context.
  *
- * \param ctx      AES context to be cleared
+ * \param ctx      The AES context to clear.
  */
 void mbedtls_aes_free( mbedtls_aes_context *ctx );
 
 /**
- * \brief          AES key schedule (encryption)
+ * \brief          This function sets the encryption key.
  *
- * \param ctx      AES context to be initialized
- * \param key      encryption key
- * \param keybits  must be 128, 192 or 256
+ * \param ctx      The AES context to initialize.
```

Comment:
Blocker: Distinguish this from `mbedtls_aes_init`. Suggestion: `The AES context to which the key should be bound.`